uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -171,7 +205,7 @@ def send_500_response(self):
         # itself (see https://github.com/encode/uvicorn/issues/920)
         self.handshake_started_event.set()
 
-    async def ws_handler(self, protocol, path):
+    async def ws_handler(self, protocol: WebSocketServerProtocol) -> Any:
```

Comment:
Curious that the signature on this method has changed.
Is that intentional?

Reply:
Yes, it is. The old signature was deprecated on `websockets`, and it causes mypy issues. I can bring it back, but I'll have to `# type: ignore[<error>]` the line.

Reply:
Whichever `websockets` *actually* calls makes sense.
What version of websockets does this change in? Are we at any risk of introducing a breakage to our users here?
How come our existing method isn't *already* broken, if it's got the wrong call signature?

Reply:
You're right. It was introduced in 10.1: https://websockets.readthedocs.io/en/stable/project/changelog.html#id2

I need to ignore this.

Reply:
I suppose switching to `protocol, path=None` would help ensure that we're okay again any of `10.0`, `10.1`, and (whenever it lands) `11.0` (?)

Reply:
I've readded it.